NitroxServer/Communication/Packets/Processors/EntityMetadataUpdateProcessor.cs

```diff
@@ -45,5 +49,22 @@ private void SendUpdateToVisiblePlayers(EntityMetadataUpdate packet, Player send
                 }
             }
```

Comment:
Just replace the whole Process code by:
```cs
if (!entityRegistry.TryGetEntityById(packet.Id, out Entity entity))
{
    Log.Error($"Entity metadata updated on an entity unknown to the server {packet.Id} {packet.NewValue.GetType()} ");
    return;
}

if (ProcessMetadata(sendingPlayer, entity, packet.NewValue))
{
    entity.Metadata = packet.NewValue;
    SendUpdateToVisiblePlayers(packet, sendingPlayer, entity);
}
```
I like TryGetEntityById because it helps avoiding the optional layer (.Value). Then we can have ProcessMetadata be kind of a security filter for our data